### DISCUSSION TITLE
chosing MP structures for multiheads finetuning

### DISCUSSION THREAD

**Discussion by triestram** [ACEsuit/mace#720] [Q&A] [answered]
Hi!
I am using  “multiheads_finetuning” for MACE-MP-0b (the medium model) and  I was wondering if there is a recommended way to replay parts of the original MP structures . 

So, for example, I am interested in a system with H, C, N and Zn, meaning that all my training data only contains these atoms. 

How should I handle the structures from the original model (mp_traj_combined.xyz). During the training? 
Should it replay:
- structures containing any atoms (even structures without H, C, N and Zn)
or
- structures only containing H, C, N and/or Zn
or
- structures containing  H, C, N and/or Zn, along with other atoms

I am using mace 0.3.8.
Thank you!

**Comment by ilyes319** [comment, marked as answer]
Hey @triestram,

The recommended way for now is to use the default selection process, that will combined the configurations in mptraj (the pre training set) that have at least two of your elements + a random selection of other configuration to diversify the replay head. 
To make your life easier, I encourage you to update to the latest main branch (0.3.9) and then just pass:
`--pt_train_file="mp"`. That will do the selection automatically.
`--num_samples_pt` can control the number of configurations to keep, the larger the less forgetting you will have but also the more expensive your finetuning becomes. The default value is a 1000 and it is safe to start with that.

**Reply by triestram** [reply]
Thanks!
